docs/pages/reference/mesheryctl-commands.md

```diff
@@ -633,6 +633,19 @@ Installation, troubleshooting and debugging of Meshery and its adapters.
           <td>{{ flag.description }}</td>
       </tr>
       {% endfor %}
+      {% assign subcommand5 = command8.subcommands.import %}
+      <tr>
+        <td><a href="{{ site.baseurl }}/reference/mesheryctl/app/import">{{ subcommand5.name }}</a></td>
+        <td></td>
+        <td>{{ subcommand5.description }}</td>
+      </tr>
+      {% for flag_hash in subcommand5.flags %}{% assign flag = flag_hash[1] %}
+        <tr>
+          <td></td>
+          <td>{{ flag.name }}</td>
+          <td>{{ flag.description }}</td>
+      </tr>
+      {% endfor %}
```

Comment:
@alphaX86 please review.

Reply:
Great job. 

We would still need to find a way for automating this though to avoid manual process